python/opm/io/deck/__init__.py

```diff
@@ -38,7 +38,11 @@ def get_item_deckitem(self):
     elif self.is_double():
         return self.get_raw(0)
     elif self.is_uda():
-        return self.get_uda(0)
+        uda_item = self.get_uda(0)
+        if uda_item.is_string():
+            return uda_item.get_string()
+        else:
+            return uda_item.get_double()
```

Comment:
This code snippet looks a lot like the definition of [`uda_value()`](https://github.com/OPM/opm-common/blob/61bdafda30fa9acb041d3a5a036a03ba2f053f11/python/opm/io/deck/__init__.py#L6-L11) above.  Is it possible to "call" that property evaluation function instead of repeating its implementation inline here?

Reply:
Yes it is, thanks. I did not notice but I have updated this now.